MELA/makefile

```diff
@@ -82,6 +86,12 @@ $(LIBRULE):	$(OBJECTS) $(MELAOBJDIR)/LinkDef_out.o | alldirs
 	echo "Linking $(LIB)"; \
 	$(LINKER) $(LINKERFLAGS) -shared $(OBJECTS) $(MELAOBJDIR)/LinkDef_out.o -o $@
 
+$(pyLIBRULE): $(pyLINKFILE) | alldirs
+	@echo "Making python"; \
+	$(LINKER) $(pyLINKFILE) $(CPPFLAGS) $(LINKERFLAGS) \
+	-O3 -Wall -shared $(OBJECTS) $(MELAOBJDIR)/LinkDef_out.o -std=c++17 -fPIC \
+	$(shell python3 -m pybind11 --includes) -o $@
```

Comment:
Also over here, conditionally compile when python 3.x is detected (I assume the bindings are python 3 specific).

Reply:
This check now happens in setup.sh - resolved.